src/main/java/com/kodilla/ecommercee/domain/Group.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Entity
+@Table(name = "products_groups")
+@Data
+@NoArgsConstructor
+public class Group {
+
+    public Group(Long id, String name) {
```

Comment:
konstruktor na duł - pod polami klasy, wogóle czy potrzebne jest to id ?

Reply:
nie jest potrzebne... Tym się jeszcze nie zajmowałem. Musiałem to tak zrobić, bo te Group pisałem na szybko, a wszędzie wymagało id.
Trzeba te id z konstruktorów wywalić wszędzie